src/CommunityToolkit.Maui.MediaElement/CommunityToolkit.Maui.MediaElement.csproj

```diff
@@ -66,4 +66,7 @@
     <PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiPackageVersion)"/>
     <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="$(MauiPackageVersion)"/>
   </ItemGroup>
+  <ItemGroup>
+    <MauiImage Include="Resources\whitefs.png" BaseSize="36,36" Resize="true" pack="true" PackagePath="." />
```

Comment:
If we are resizing this would it be better as an SVG instead?

Reply:
Thanks @bijington for the feedback. The image has been replaced by the SVG version. Marked as resolved.